DelegationStation/Services/GraphService.cs

```diff
@@ -31,18 +32,18 @@ public GraphService(IConfiguration configuration, ILogger<GraphService> logger)
             var scopes = new string[] { $"{graphEndpoint}.default" };
             string baseUrl = graphEndpoint + "v1.0";
 
-            var certConfig = configuration.GetSection("AzureAd:ClientCertificates").GetChildren().FirstOrDefault();
+            var certDN = configuration.GetSection("AzureAd:ClientCertificates:CertificateDistinguishedName").Value;
```

Comment:
Will this have issues if more than one certificate is specified?

Reply:
No -- confirmed we don't need this right now, documented in ADO to resolve in the future